tests/testthat/_snaps/expect-condition.md

```diff
@@ -35,12 +35,35 @@
       expect_condition(stop("Hi!"), foo = "bar")
     Condition
       Error in `expect_condition()`:
-      ! Can't specify `...` without `pattern`.
+      ! Can't supply `...` unless `regexp` is set.
+      * Unused arguments: `foo`.
+      i Did you mean to use `regexp` so `...` is passed to `grepl()`?
+    Code
+      expect_condition(stop("Hi!"), , , "bar")
+    Condition
+      Error in `expect_condition()`:
+      ! Can't supply `...` unless `regexp` is set.
```

Comment:
I'm not sure how actionable this is, OTOH, I think it's unlikely to arise organically anyway (this is very strange code). Not worth spending more time on it IMO.